src/Button.vue

```diff
@@ -131,23 +136,23 @@ export default {
     },
 
     colorChecked () {
-      let { color } = this
-
-      if (typeof color !== 'object') {
-        return color || constants.colorChecked
-      }
-
-      return contains(color, 'checked')
-        ? color.checked
-        : constants.colorChecked
+      let { color, cssColors } = this
+      return cssColors
+        ? undefined
+        : typeof color !== 'object'
+          ? color || constants.colorChecked
+          : contains(color, 'checked')
+            ? color.checked
+            : constants.colorChecked
```

Comment:
Chaining ternary operators is an anti-pattern in javascript. Never use more than one.

Reply:
Who says so? But sure, I'll change it.